recipe/patches/0005-Update-installation-of-fsts-headers.patch

```diff
@@ -0,0 +1,38 @@
+From dc4fbab1500ec01cddcef5ec01b29f942fd89ff4 Mon Sep 17 00:00:00 2001
```

Comment:
@kylebgorman here's the patch for the headers if you want to use it for upstream

Reply:
FYI I think you want to patch `Makefile.am` not `.in`. `.in` files are autogenerated from `.am` files.

Reply:
Though you may have to fix `.am`, run the autoconf stuff, and then catch the autogenerated `.in` file to make its patch. Annoying.

Reply:
Ahh ok, not too familiar with the autotools build stuff, but I'll play around and see if there's a better place to patch it.

Reply:
Ok, the better version is merged in and tests passing!